README.md

```diff
@@ -30,12 +30,14 @@ to passenger's interaction with the bus directly through their phone.
 Please refer below pages  for a complete tutorial of the system; 
 - [User manual](https://github.com/DIT112-V21/group-07/wiki/User-Manual)
 - [Project charter](https://github.com/DIT112-V21/group-07/wiki/Project-Charter)
-- [Wiki pages](https://github.com/DIT112-V21/group-07/wiki/Pathfinder's-Wiki#welcome-to-the-pathfinders-wiki)
+- [Wiki pages](https://github.com/DIT112-V21/group-07/wiki)
 
 ***
 
 ### App Installation
 
+Installation guidance are provided in detail on the Wiki pages below.
+
 - To install the application, please proceed to [user installation](https://github.com/DIT112-V21/group-07/wiki/User-installation-&-setup-guidance).
```

Comment:
Do we need User Installation vs **Developer Installation**? If someone's a developer, we should advise on how to fork the repo for them to contribute or some other strategy. Perhaps all necessary set up instructions are covered under **User Installation**, rendering the former redundant? Just a thought

Reply:
Maybe, developer installation is on Wiki. If you have the idea, you can just edit it